packages/zowe-explorer/src/Profiles.ts

```diff
@@ -557,6 +572,50 @@ export class Profiles extends ProfilesCache {
         return profileType;
     }
 
+    public async createZoweSchema(zoweFileProvider: IZoweTree<IZoweTreeNode>) {
+        try {
+            ImperativeConfig.instance.loadedConfig = {
+                defaultHome: path.join(os.homedir(), ".zowe"),
+                envVariablePrefix: "ZOWE",
+            };
+
+            // If there are no workspaces already opened, default to the cliHome
+            const rootPath = vscode.workspace.workspaceFolders?.[0].uri.fsPath || ImperativeConfig.instance.cliHome;
+            // true-case-path may or may not be needed. Better safe than sorry : )
+            const config = await Config.load("zowe", { projectDir: trueCasePathSync(rootPath) });
+
+            const impConfig: IImperativeConfig = zowe.getImperativeConfig();
+            const knownCliConfig: ICommandProfileTypeConfiguration[] = impConfig.profiles;
+            knownCliConfig.push(impConfig.baseProfile);
+            config.setSchema(ConfigSchema.buildSchema(knownCliConfig));
```

Comment:
Is this going to overwrite the existing schema file?
If so, what will happen to the existing profile type definitions (e.g. DB2, IMS...)?

Reply:
Hey @zFernand0 . Thanks for taking a look. I tried it with CICS, DB2, and other profile types and they were not overwritten. The "base" definition was inserted in the middle, but the rest remained untouched.